SimPEG/EM/Base.py

```diff
@@ -154,6 +197,23 @@ def MeMuI(self):
             self._MeMuI = self.mesh.getEdgeInnerProduct(self.mu, invMat=True)
         return self._MeMuI
 
+    # TODO: This should take a vector
+    def MeMuIDeriv(self, u):
+        """
+        Derivative of :code:`MeSigma` with respect to the model
+        """
+        # TODO: only works for diagonal tensors. getEdgeInnerProductDeriv,
+        #       invMat=True should be implemented in SimPEG
+
+        if (
+            getattr(self, 'muMap', None) is not None or
```

Comment:
right now, the mu inversion is only implemented in the frequency domain, so if you were to set up a time domain problem, the attribute currently does not exist. 
Perhaps though, getting a 'muMap' does not exsist error would be ok in the mean-time?